benchmarks/README.md

```diff
@@ -0,0 +1,24 @@
+# dpctl benchmarks
+
+Benchmarking dpctl using Airspeed Velocity
+Read more about ASV [here](https://asv.readthedocs.io/en/stable/index.html)
+
+## Usage
+This was made for use within our CI in mind. As such, you will see the `asv.conf.json` is minimal without any environmental information supplied.
+The expectation is for users to execute `asv run` with an existing environment.
+
+As such, you should have conda or mamba installed, and create a `dpctl-benchmarking` environment as so:
+```
+conda create --name dpctl-benchmarking python=$PYTHON_VERSION dpctl asv libmambapy conda dpcpp_linux-64 --override-channels \
+        -c https://software.repos.intel.com/python/conda \
+        -c conda-forge -y
+```
```

Comment:
I think this should be re-written from more of a user perspective, as in, I think we should encourage users to run these benchmarks locally, regardless of our intention to use in CI.

The `conda` command also only handles case where the user does not have oneAPI locally installed. We can just refer to creating an environment using the [installation instructions](https://intelpython.github.io/dpctl/latest/beginners_guides/installation.html#dpctl-installation)

then instruct to install addition dependencies (asv, libmamabapy)